src/libs/ReportUtils.ts

```diff
@@ -8034,10 +8034,17 @@ function canLeaveChat(report: OnyxEntry<Report>, policy: OnyxEntry<Policy>): boo
     return (isChatThread(report) && !!getReportNotificationPreference(report)) || isUserCreatedPolicyRoom(report) || isNonAdminOrOwnerOfPolicyExpenseChat(report, policy);
 }
 
-function getReportActionActorAccountID(reportAction: OnyxInputOrEntry<ReportAction>, iouReport: OnyxInputOrEntry<Report> | undefined): number | undefined {
+function getReportActionActorAccountID(
+    reportAction: OnyxInputOrEntry<ReportAction>,
+    iouReport: OnyxInputOrEntry<Report> | undefined,
+    report: OnyxInputOrEntry<Report> | undefined,
+): number | undefined {
     switch (reportAction?.actionName) {
-        case CONST.REPORT.ACTIONS.TYPE.REPORT_PREVIEW:
-            return !isEmptyObject(iouReport) ? iouReport.managerID : reportAction?.childManagerAccountID;
+        case CONST.REPORT.ACTIONS.TYPE.REPORT_PREVIEW: {
+            const ownerAccountID = iouReport?.ownerAccountID ?? reportAction?.childOwnerAccountID;
+            const actorAccountID = iouReport?.managerID ?? reportAction?.childManagerAccountID;
+            return isPolicyExpenseChat(report) ? ownerAccountID : actorAccountID;
+        }
```

Comment:
I'm trying to understand this logic as I need to fix a similar issue over [here](https://github.com/Expensify/App/issues/55906). 

Could you give me an example of when `actorAccountID` (`managerID`) should be used for a report action?

Reply:
In IOU's report preview
<img width="500" alt="Screenshot 2025-01-30 at 5 38 39 PM" src="https://github.com/user-attachments/assets/b6a1a8aa-0862-435d-9105-2059a6e57156" />

Otherwise it would look like this
<img width="500" alt="Screenshot 2025-01-30 at 5 38 53 PM" src="https://github.com/user-attachments/assets/8dcde005-634d-429d-b8e0-fbe816d5f408" />